src/styles/search.css

```diff
@@ -0,0 +1,20 @@
+.search {
+  color: black;
+}
+
+.search-form {
+  color: black;
+  text-align: center;
+}
+
+.search-input {
+  color: black;
+}
+.search-button {
+  color: black;
+}
+
+.search-button:hover {
+  background-color: lightgray;
+  cursor: pointer;
+}
```

Comment:
As above this could be condensed a bit
```
.search,
.search-input,
.search-button {
  color: black;
}

.search-form {
  color: black;
  text-align: center;
}

.search-button:hover {
  background-color: lightgray;
  cursor: pointer;
}
```